core.go

```diff
@@ -43,6 +44,7 @@ func (b *Core) PopulateUsing(paths storage.PathList) error {
 
 		}
 	}
+	fmt.Printf("Finished populating credentials\n")
```

Comment:
Eventually, the code will need to be moved around so that this can be written to the server logger.